data/sql/updates/pending_db_world/rev_1643656464415474000.sql

```diff
@@ -0,0 +1,78 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1643656464415474000');
+
+-- All Blackwing Technician's spawns
+DELETE FROM `creature` WHERE `id1` IN (13996) AND `guid` IN(84533, 84534, 84535, 84536, 84537, 84538, 84539, 84540, 84544, 84545, 84547, 84549, 84550, 84551, 84552, 84554, 84555, 84556, 84562, 84563, 84564, 84565, 84566, 84567, 84568, 84569, 84570, 84571, 84602, 84617, 84618, 84619, 84623, 84624, 84654, 84655, 84688, 84689, 84691, 84692, 84759, 84760, 84761, 84762, 84763, 85609, 85611, 85612, 85613, 85616, 85620, 85621, 85622, 85623, 85625, 85760, 85764, 85766, 85767, 85768, 85772, 85773, 85776, 85777, 85778, 85779);
```

Comment:
the deleted guids are not matching the guids that are being inserted

Reply:
Not yet resolved